templates/galaxy/config/job_conf.yml

```diff
@@ -588,6 +588,9 @@ galaxy_jobconf:
     - id: usegpu
       names: [gpu]
   limits:
+    - type: "destination_user_concurrent_jobs"
+      id: "condor_upload"
+      value: "5"
```

Comment:
Lets do 10 here. We want to limit users that upload 100 of files etc at once ...